panel/widgets/input.py

```diff
@@ -227,6 +227,8 @@ def save(self, filename):
                     f.write(val)
             else:
                 fn.write(val)
+                if hasattr(fn, "seek"):
```

Comment:
I've only seen `BytestIO` as `fn`. But maybe others need to be supported as well?